e2e/e2etests/e2etests.go

```diff
@@ -158,6 +159,11 @@ var AllE2ETests = []runner.E2ETest{
 		"withdraw Ether from ZEVM",
 		TestEtherWithdraw,
 	},
+	{
+		TestEtherWithdrawRestrictedName,
+		"withdraw Ether from ZEVM to restricted address",
+		TestEtherWithdrawRestricted,
```

Comment:
Can we do the same with Bitcoin and add a separate test for restricted address?
The reason for this is that these tests can be run on live network and therefore we want to keep those minimal in case we just want to test a very specific feature on a live network

Reply:
https://github.com/zeta-chain/node/pull/1791/commits/7c4bebe80d93b1474ca1913ddd24694e9de9c1d1